src/System.Memory/src/System.Memory.csproj

```diff
@@ -0,0 +1,55 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum);netcoreapp2.1</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <StrongNameKeyId>Open</StrongNameKeyId>
+    <Nullable>disable</Nullable>
+    <IsPackable>false</IsPackable>
+    <VersionPrefix>4.5.5</VersionPrefix>
+    <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.6.0</VersionPrefix>
+    <AssemblyVersion>4.0.1.2</AssemblyVersion><!-- NO-INCREMENT: This version is frozen for .NET Standard and .NETCoreApp because the assembly ships inbox. -->
+    <AssemblyVersion Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.0.1.2</AssemblyVersion>
+    <AssemblyVersion Condition="'$(IsPackable)' == 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.0.2.0</AssemblyVersion>
+    <PackageValidationBaselineVersion>4.5.5</PackageValidationBaselineVersion>
+    <IsPlaceholderTargetFramework Condition="'$(TargetFramework)' == 'netcoreapp2.1'">true</IsPlaceholderTargetFramework>
+    <DefineConstants Condition="'$(IsPlaceholderTargetFramework)' != 'true'">$(DefineConstants);NETSTANDARD</DefineConstants>
+    <ILLinkClearInitLocals>true</ILLinkClearInitLocals>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(IsPlaceholderTargetFramework)' == 'true'">
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
```

Comment:
Same feedback as in System.Buffers, this should be the default setting and moved into the placeholder targets file.